File: dal/src/main/java/com/bykea/pk/partner/dal/source/remote/response/GetDriverProfile.java
package com.bykea.pk.partner.dal.source.remote.response;

import com.bykea.pk.partner.dal.source.remote.data.PersonalInfoData;

public class GetDriverProfile extends BaseResponse {

    private PersonalInfoData data;

    public PersonalInfoData getData() {
        return data;
    }

    public void setData(PersonalInfoData data) {
        this.data = data;
    }
}

File: dal/src/main/java/com/bykea/pk/partner/dal/source/remote/response/GetDriverProfile.kt
package com.bykea.pk.partner.dal.source.remote.response

import com.bykea.pk.partner.dal.source.remote.data.PersonalInfoData
import com.google.gson.annotations.SerializedName

data class GetDriverProfile(
        @SerializedName("data") var data: PersonalInfoData?
) : BaseResponse() {
}
